packages/consumption/src/modules/notifications/NotificationsController.ts

```diff
@@ -34,7 +34,7 @@ export class NotificationsController extends ConsumptionBaseController {
     }
 
     public async sent(message: Message): Promise<LocalNotification> {
-        if (!message.isOwn) throw new Error("Cannot send a Notification from a foreign message.");
+        if (!message.isOwn) throw new Error("Cannot save a LocalNotification with status 'Sent' from a received Message.");
```

Comment:
I think you forgot to undo stuff here

Reply:
I only changed the Message because the Message before wasn't right. See the discussions before. Or what do you mean?

Reply:
Ok, maybe I am just confused. What do you think about: `Cannot mark a LocalNotification as sent from a received Message.`

Reply:
Okay so it would be an error message analogous to the one in the handleMessageReceivedEvent.

Reply:
yep!